src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -1475,4 +1481,27 @@ private void ComputePackageExclusions()
             }
         }
     }
+
+    internal static class NuGetTargetFrameworkUtils
+    {
+        public static NuGetFramework GetTargetFramework(string targetFrameworkMoniker, string targetPlatformMoniker)
+        {
+            var targetFramework = NuGetUtils.ParseFrameworkName(targetFrameworkMoniker);
+            if (targetPlatformMoniker != null && !targetPlatformMoniker.Equals(string.Empty))
+            {
+                try
+                {
+                    var targetPlatform = NuGetUtils.ParseFrameworkName(targetPlatformMoniker);
+                    var constructedFramework = new NuGetFramework(targetFramework.Framework, targetFramework.Version, targetPlatform.Framework, targetPlatform.Version);
+
+                    return constructedFramework;
+                }
+                catch
```

Comment:
We should generally avoid generic catch blocks like this.  What exception are we expecting?

Reply:
I was seeing an exception with invalid target platform versions without this change, which was odd because I couldn't repro it locally. In general I agree, but as this is a work around and should be removed soon I think it should be okay.